lms/djangoapps/grades/rest_api/v1/tests/test_views.py

```diff
@@ -257,3 +260,293 @@ def test_staff_can_get_all_grades(self):
+                                     'percent': 0.0},
+                                    {'category': 'Lab',
+                                     'detail': 'Lab 12 Unreleased - 0% (?/?)',
+                                     'label': 'Lab 12',
+                                     'mark': {'detail': 'The lowest 2 Lab scores are dropped.'},  # nopep8 pylint: disable=C0301
+                                     'percent': 0.0},
+                                    {'category': 'Lab',
+                                     'detail': 'Lab Average = 0%',
+                                     'label': 'Lab Avg',
+                                     'percent': 0.0,
+                                     'prominent': True},
+                                    {'category': 'Midterm Exam',
+                                     'detail': 'Midterm Exam = 0%',
+                                     'label': 'Midterm',
+                                     'percent': 0.0,
+                                     'prominent': True},
+                                    {'category': 'Final Exam',
+                                     'detail': 'Final Exam = 0%',
+                                     'label': 'Final',
+                                     'percent': 0.0,
+                                     'prominent': True}]
+
+    def get_url(self, query_params=None):
+        """
+        Helper function to create the url
+        """
+        base_url = reverse(
+            self.namespaced_url,
+        )
+        if query_params:
+            base_url = f'{base_url}?{query_params}'
+        return base_url
+
+    def test_anonymous(self):
+        resp = self.client.get(self.get_url())
+        assert resp.status_code == status.HTTP_401_UNAUTHORIZED
+
+    def test_student(self):
+        self.client.login(username=self.student.username, password=self.password)
+        resp = self.client.get(self.get_url())
+        assert resp.status_code == status.HTTP_403_FORBIDDEN
+
+    def test_course_does_not_exist(self):
+        self.client.login(username=self.global_staff.username, password=self.password)
+        resp = self.client.get(
+            self.get_url(
+                urlencode({'course_id': 'course-v1:MITx+8.MechCX+2014_T1'})
+            )
+        )
+        expected_data = OrderedDict([
+            ('next', None),
+            ('previous', None),
+            ('results', [])
+        ])
+        assert resp.status_code == status.HTTP_200_OK
+        assert expected_data == resp.data
+
+    def test_course_no_enrollments(self):
+        self.client.login(username=self.global_staff.username, password=self.password)
+        resp = self.client.get(
+            self.get_url(urlencode({'course_id': self.empty_course.id}))
+        )
+        assert resp.status_code == status.HTTP_200_OK
+        expected_data = OrderedDict([
+            ('next', None),
+            ('previous', None),
+            ('results', []),
+        ])
```

Comment:
```suggestion
        expected_data = OrderedDict(
            [
                ('next', None),
                ('previous', None),
                ('results', []),
            ]
        )
```

Nit: suggested a more readable formatting.